cluster/local/util.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
this needs the copyright boilerplate from `master/hack/boilerplate/boilerplate.sh.txt`.